rdmd_test.d

```diff
@@ -487,8 +487,9 @@ void runTests(string rdmdApp, string compiler, string model)
     }
 
     /* [REG2.072.0] pragma(lib) is broken with rdmd: https://issues.dlang.org/show_bug.cgi?id=16978 */
+    /* GDC does not support this pragma(lib), so disable when test compiler is gdmd */
 
-    version (linux)
+    version (linux) if (compiler.baseName != "gdmd")
```

Comment:
It needs two issues -- one for GDC (w.r.t. the need for `pragma(lib)` support) and one for rdmd that references that.

@ibuclaw is there an existing `pragma(lib)` issue open for GDC?  The only one I could find by searching was this: https://bugzilla.gdcproject.org/show_bug.cgi?id=153

... which is not really the core issue.

Reply:
OK, I'll tweak.  BTW it should also be possible to remove the double `{{` and `}}` inside an `if` block (they are there with `version (linux)` because the `version (linux) { ... }` does not create a scope around the tests).